tests/strategies/filter/test_crop_filter.py

```diff
@@ -0,0 +1,50 @@
+"""Tests the crop filter strategy."""
+from shutil import copy
+
+
+def test_crop_filter(tmp_path):
```

Comment:
```suggestion
from shutil import copy
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from pathlib import Path


def test_crop_filter(tmp_path: Path):
```

Reply:
Doing this ends in a
`NameError: name 'Path' is not defined`
on the line
`def test_crop_filter(tmp_path: Path):`

Reply:
Right.. sorry.. should be `tmp_path: "Path"`